infrastructure/templates/datafactory/components/db-maintenance-template.json

```diff
@@ -130,7 +130,7 @@
               },
               "body": {
                 "channel": "[parameters('slackAlertsChannel')]",
-                "text": "Data Factory Success!",
+                "text": "Reindexing complete or Data Factory timeout at 16hrs - stored procedure may still be running!",
```

Comment:
If you make this change we are going to question whether it's still running or not every time it's reported, or at least we would need to look at the run time to see if it's 16 hours or not, to determine if it was successful or a timeout.

If it completes after 16 hours, the data factory task should timeout at 16 hours and it should trigger the `Report failure` activity, so I think the text on L355 should be changed to report something like `Data Factory Failure! Stored procedure may still be running`, rather than this change here.

If it completes successfully within 10 hours it would report `Data Factory Success!`.

If it completes somewhere between 10 hours and 16 hours because it began stopping, it would either report `Data Factory Success` or `Data Factory Failure! Stored procedure may still be running` depending on whether you make the change suggested in my other comment about the stored procedure, which will affect whether it's considered a success or failure.